src/web/app/src/components/DependenciesTable/Row.tsx

```diff
@@ -19,9 +19,11 @@ import { dependencyDiscoveryUrl } from '../../config';
 const useStyles = makeStyles((theme) => ({
   root: {
     '& .MuiTableCell-body': {
-      fontSize: '14px',
+      fontFamily: 'Spartan',
+      fontSize: '16px',
       color: theme.palette.text.secondary,
       fontWeight: 700,
+      borderBottom: '0.01em solid #909090',
```

Comment:
I believe your style is not applying
![image](https://user-images.githubusercontent.com/58233223/198403045-8aa4e6c3-4057-4ad7-8ba0-03ffb5e3b6e0.png)

Reply:
I checked the vercel deployment on Firefox and Google Chrome, and the style seems to be applying on my device. Could this be due to different browsers or something else? 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/67077705/198404410-6e9f3028-f0b9-4fa4-9242-641c8d714c2e.png">

Reply:
Woops sorry I mean the `dependencyInfoCell`

Reply:
I checked all the three cells (per dependencyTable Row), and it appears to be applying to me. Could you please share a screenshot of the element you were inspecting?

Reply:
![image](https://user-images.githubusercontent.com/58233223/198843536-6e5211f7-136e-4a7f-a0f7-0ed3fac1aab0.png)
Running on chrome Version 106.0.5249.121 (Official Build) (64-bit)


```suggestion
      borderBottom: '0.01em solid #909090 !important', 
```
I believe can fix it.

Let's use `1px` instead of `0.01em`. I feel there is no difference visual.

Reply:
Yep, that should fix it. Thanks

Though I had to use `0.01em` instead of `1px` since the browser will render the border width slightly different for every border with `px`. This is what caused [this color difference](https://github.com/Seneca-CDOT/telescope/pull/3702#discussion_r992924326)